### PR TITLE
CHICKEN SANDWICH PLEASE!

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @gusshawstewart
- [ ] @ztsygan
- [ ] @lpoolma1
- [ ] @bseeger
- [ ] @jhujasonw
- [ ] @reinacmurray
- [ ] @derekbelrose
- [ ] @jgara
- [ ] @jaredgalanis
- [ ] @Bonni
- [ ] @jwang40
- [ ] @akatz2
- [ ] @EvanBolt
- [ ] @cyork
- [ ] @jhu-alistair
- [ ] @michaelsainz
- [ ] @acornwe3
- [ ] @parkerbxyz

### PR DESCRIPTION
**Pull request by jhujasonw**
TL;DR 
# CHICKEN SANDWICH

### Done
- closes #2 
- created first PR

### TODO
- [ ] merge pr
- [ ] delete branch
- [ ] send Jason a chicken sandwich

cc: @derekbelrose @jgara @acornwe3 @gusshawstewart

### CODE REVIEW

**Comment by derekbelrose** [comment]
LGTM!

**Comment by bseeger** [comment]
I want a chicken sammy now too.

**Comment by jgara** [comment]
mmmm. chicken...

File: _posts/0000-01-05-jhujasonw.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Chick-Fil-A chicken Sandwiches are awesome! Can someone deliver one to me please?
```
Comment (jgara) [inline _posts/0000-01-05-jhujasonw.md:6]: Ah, but alas, I do _not_ have one better!

**Pull request merged**